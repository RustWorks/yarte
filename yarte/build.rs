#[path = "src/recompile.rs"]
mod recompile;

fn main() {
    recompile::when_changed();
}
