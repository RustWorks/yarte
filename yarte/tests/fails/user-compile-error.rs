use yarte::Template;

#[derive(Template)]
#[template(src = "{{#if true.is_some() }}
{{$ \"OMG! true is some\" }} {{/if }}")]
struct Test{
    foo: usize
}

fn main() {}
