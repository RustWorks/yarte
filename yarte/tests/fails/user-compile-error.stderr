error: proc-macro derive panicked
 --> $DIR/user-compile-error.rs:3:10
  |
3 | #[derive(Template)]
  |          ^^^^^^^^
  |
  = help: message: error
           --> $DIR/Test.hbs:1:7
            |
          1 | {{#if true.is_some() }}
            |       ^^^^^^^^^^^^^^ Compile error: OMG! true is some
            |
